flipperdevices/flipperzero-firmware
# [FL-2480] Use SysTick as the main OS timer

# What's new
- Use SysTick as the main timer for FreeRTOS scheduler
- Use LPTIM2 as the wakeup timer
- Fix unwanted LPTIM2 IRQ handler call on sleep mode exit
- Slightly improve LPTIM2 accuracy
- Deep sleep is **NOT** in this PR's scope

# Verification
- Use Flipper as usual. The animations should have their normal speed. The device should not hang up.
 - Run `FURI_HAL_OS_DEBUG=1 make debug` and verify that the low period length on PA7 is equal to `expected_idle_ticks` within 1ms or less.

# Checklist (For Reviewer)
